public/app/features/provisioning/SyncRepository.tsx

```diff
@@ -40,7 +40,7 @@ export function SyncRepository({ repository }: Props) {
     if (!name) {
       return;
     }
-    syncResource({ name, body: { complete: true } });
+    syncResource({ name, body: {} }); // will queue a full resync job
```

Comment:
can you display the summary you created and the progress in the ugly UI as this PR is keep the progress while running the sync job?

Reply:
that sounds great -- but lets do that in a new PR... this has already got so many parts